pkg/pipeline/CiCdPipelineOrchestrator.go

```diff
@@ -78,6 +79,10 @@ type CiCdPipelineOrchestrator interface {
 	CheckStringMatchRegex(regex string, value string) bool
 	CreateEcrRepo(dockerRepository, AWSRegion, AWSAccessKeyId, AWSSecretAccessKey string) error
 	GetCdPipelinesForEnv(envId int, requestedAppIds []int) (cdPipelines *bean.CdPipelines, err error)
+
+	InitiateMigrationOfStageScriptsToPipelineStageSteps(cdPipeline *bean.CDPipelineConfigObject) (*bean.CDPipelineConfigObject, error)
+	StageStepsToCdStageAdapter(deployStage *bean2.PipelineStageDto) (*bean.CdStage, error)
+	CheckForVersionAndCreatePreAndPostStagePayload(cdPipeline *bean.CDPipelineConfigObject, version string, appId int) (*bean.CDPipelineConfigObject, error)
```

Comment:
Check for version seems redundant. This can be explained in the implementation with a comment :)

Reply:
done